app/views/merchant/dashboard/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Merchant Dashboard</h1>
+<br>
+<h2>User: <%= @user.name %></h2>
```

Comment:
I love that every thing is working! just marking all the instances for refactor for the 'nil? == false'